Lib/pyclbr.py

```diff
@@ -58,31 +59,36 @@ def __init__(self, module, name, file, lineno, parent):
         self.lineno = lineno
         self.parent = parent
         self.children = {}
+        if parent is not None:
+            parent._addchild(name, self)
```

Comment:
Child nesting was previously handled in _nest_function, and _nest_class, which were only called when parent was not None.  If nesting is moved into _Object, I believe those functions and _addchild are not needed and can be deleted.  Even if _addchild were needed, in-lining the subscription would be faster.  I could have done this when I added function nesting for 3.7, but I likely made minimal changes and never re-examined the result until now.  If you want me to try out the change, let me know.


```suggestion
        if parent is not None:
            parent.children[name] = self
```